Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: run-explorer run-tests run-linters build-ui build-python build-docker run-docker compose-up
 
-version="0.59.0"
```

Comment:
Does this need a bump as we are not releasing this feature yet? Just adding the code.

Reply:
Agree !